src/common/components/select-repositories-page/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchUserRepositories } from '../../actions/repositories';
+import SelectRepositoryList from '../select-repository-list';
+import { HeadingTwo, HeadingThree } from '../vanilla/heading';
+import { CardHighlighted } from '../vanilla/card';
+
+class SelectRepositoriesPage extends Component {
+  componentDidMount() {
+    const { authenticated } = this.props.auth;
+
+    if (authenticated) {
+      this.props.dispatch(fetchUserRepositories());
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <HeadingTwo>
+          All your repositories have been successfully added to build.snapcraft.io
```

Comment:
@cjwatson: It was the most recent copy in the design spec document at the time of writing.